Documentation/ceph-filesystem.md

```diff
@@ -128,6 +128,25 @@ Create the storage class.
 kubectl create -f cluster/examples/kubernetes/ceph/csi/cephfs/storageclass.yaml
 ```
 
+## Mirroring
```

Comment:
Why do we have both the `mirror.enabled` flag and the CRD? I don't see why we need both.

Reply:
The CRD deploys the cephfs-mirror daemon, the CephFilesystem setting, turns on mirroring on that filesystem.

Reply:
will this enable mirroring of complete filesystem by default? do we have an option to enable per subvolume not per filesystem like we do for RBD?

Reply:
Yes, but just like rbd-mirroring must be first enabled on the pool and then per-image, we have to enable the mirroring on the fs and then we can do snapshot per-directory using subcommand like: `ceph fs snapshot mirror add <fs> <path>`.